common/app/common/configuration.scala

```diff
@@ -474,6 +474,21 @@ class GuardianConfiguration extends Logging {
     lazy val config = configuration.getMandatoryStringProperty("front.config")
   }
 
+  object targeting {
+    lazy val campaignsUrl = configuration.getStringProperty("targeting.campaignsUrl")
+
+    // Total number of rules allowed per campaign, any campaigns with more than one rule will be filtered
+    lazy val ruleLimit: Int =
+      Try(configuration.getStringProperty("targeting.ruleLimit").get.toInt)
```

Comment:
do we even need the limits to be configured (are they different between code and prod?) or can they just be vals in the code?